lib/views/users/new.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Please sign up</h2>
+
+<% if flash[:errors] && !flash[:errors].empty? %>
+  <div id='errors'><% flash[:errors].each do |error| %>
+      <li><%= error %></li>
+    <% end %>
+<% end %>
+
+<form action="/users" method="post" id="submit-forms">
+	Name: <input type="text" name="name" id="entry-box"><br>
+	Username: <input type="text" name="username" id="entry-box"><br>
+	Email: <input type="text" name="email" id="entry-box"><br>
+	Password: <input type="password" name="password" id="entry-box"><br>
+	Confirm Password: <input type="password" name="password_confirmation" id="entry-box"><br>
```

Comment:
See the stuff about `<label>` earlier. This stuff will be all weird for screen readers.